src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/alert_request.cc

```diff
@@ -74,13 +74,12 @@ AlertRequest::~AlertRequest() {}
 bool AlertRequest::Init() {
   /* Timeout in milliseconds.
      If omitted a standard value of 10000 milliseconds is used.*/
+  uint32_t request_timeout_ = 0;
   if ((*message_)[strings::msg_params].keyExists(strings::duration)) {
-    default_timeout_ =
+    request_timeout_ =
         (*message_)[strings::msg_params][strings::duration].asUInt();
-  } else {
-    const int32_t def_value = 5000;
-    default_timeout_ = def_value;
   }
+  default_timeout_ = default_timeout_ + request_timeout_;
```

Comment:
@ValeriiMalkov use +=

Reply:
@AKalinich-Luxoft Fixed in commit 76a1afadfa7423d3c12719ea6f4e49a302c04395